pennylane/variable.py

```diff
@@ -156,27 +156,31 @@ def val(self):
         values = VariableRef.kwarg_values[self.name]
         return values[self.idx] * self.mult
 
-    def render(self, show_name_only=False):
+    def render(self):
         """Returns a string representation of the VariableRef.
 
-        Args:
-            show_name_only (bool, optional): Render the name instead of the value.
-
         Returns:
             str: A string representation of the VariableRef
         """
-        if not show_name_only:
-            if self.is_kwarg and VariableRef.kwarg_values and self.name in VariableRef.kwarg_values:
-                return str(round(self.val, 3))
+        # TODO: re-enable the string representation of the VariableRer and
+        # add the show_name_only parameter
 
-            if (
+        '''
+        if not show_name_only:
+            if self.is_kwarg and VariableRef.kwarg_values and self.name in VariableRef.kwarg_values or
+            (
                 not self.is_kwarg
                 and VariableRef.positional_arg_values is not None
                 and len(VariableRef.positional_arg_values) > self.idx
             ):
                 return str(round(self.val, 3))
 
-        if self.mult != 1:
-            return "{}*{}".format(str(round(self.mult, 3)), self.name)
+        if :
+        '''
```

Comment:
I would make this a multiline comment, rather than a multiline string statement (it also allows you to remove the pylint check above)

Reply:
Also, how come the string above doesn't match the previous code? Ideally, we should leave as much of the previous code as-is, just commented out (and with instructions for what to do when the circuit drawer functionality is returned).